src/isIsogram.test.js

```diff
@@ -6,4 +6,39 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+  
+  it('should return true for an empty string', () => {
+    const result = isIsogram('');
+
+    expect(result).toBe(true);
+  });
+
+  it('should return true for a lowercase isogram', () => {
+    const result = isIsogram('playgrounds');
+
+    expect(result).toBe(true);
+  });
+
+  it('should return false for a mixed-case word', () => {
+    const result = isIsogram('Adam');
+
+    expect(result).toBe(false);
+  });
+
+  it('should return false for a word with repeating letters', () => {
+    const result = isIsogram('look');
+
+    expect(result).toBe(false);
+  });
+
+  it('should handle accented characters', () => {
+    const result = isIsogram('résum');
```

Comment:
The test case 'should handle accented characters' is unclear and may not be necessary. If accented characters are to be considered different from their unaccented counterparts, the test case is valid. However, if accented characters are to be considered the same as their unaccented counterparts, this test case could fail and should be revised or removed.